### ISSUE TITLE
"retry_interval" value is not correctly handled

### ISSUE THREAD

**Issue by banderas07** [naparuba/shinken#383]
For hosts checks I have the following config:
      " max_check_attempts              10
        check_interval                  1
        check_period                    24x7
        retry_interval                  1"

So I would expect that Shinken will trigger an alert after 10 failed consecutive checks and the time interval between the checks to be 1min(60 sec). I didn't touch the "interval_length" parameter so it has the default value, which is hardcoded I guess because it is not present in any shinken cfg file.

Extract from shinken.log:
[Thu Jun 14 13:49:20 2012] HOST ALERT: 10.2.0.2;DOWN;SOFT;1;PING CRITICAL - Packet loss = 100%
[Thu Jun 14 13:49:26 2012] HOST ALERT: 10.2.0.2;DOWN;SOFT;2;CRITICAL - Host Unreachable (10.2.0.2)
[Thu Jun 14 13:49:37 2012] HOST ALERT: 10.2.0.2;DOWN;SOFT;3;CRITICAL - Host Unreachable (10.2.0.2)
[Thu Jun 14 13:49:58 2012] HOST ALERT: 10.2.0.2;DOWN;SOFT;4;CRITICAL - Host Unreachable (10.2.0.2)
[Thu Jun 14 13:50:07 2012] HOST ALERT: 10.2.0.2;DOWN;SOFT;5;CRITICAL - Host Unreachable (10.2.0.2)
[Thu Jun 14 13:50:27 2012] HOST ALERT: 10.2.0.2;DOWN;SOFT;6;CRITICAL - Host Unreachable (10.2.0.2)
[Thu Jun 14 13:50:34 2012] HOST ALERT: 10.2.0.2;DOWN;SOFT;7;CRITICAL - Host Unreachable (10.2.0.2)
[Thu Jun 14 13:50:47 2012] HOST ALERT: 10.2.0.2;DOWN;SOFT;8;CRITICAL - Host Unreachable (10.2.0.2)
[Thu Jun 14 13:50:59 2012] HOST ALERT: 10.2.0.2;DOWN;SOFT;9;CRITICAL - Host Unreachable (10.2.0.2)

As you can see the "retry_interval" between the checks is never 60 seconds :(
